djangopeople/djangopeople/migrations/0002_load_fixtures.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.core.management import call_command
+from django.db import migrations
+
+
+def forwards_func(apps, schema_editor):
+    call_command('loaddata', 'initial', verbosity=0)
```

Comment:
@MarkusH -- I know [this isn't ideal](https://code.djangoproject.com/ticket/24778) but it seems to work for now. Is it okay or do you have some other suggestion?